Se agrega TMA MID

wichoesparza opened the pull request:
Primera vez que agrego sector para TWR. Favor de revisar.

wichoesparza committed:
Se agrega TMA MID

Primera vez que agrego sector para TWR. Favor de revisar.

hatoriganzo commented on Positions.xml:
```diff
@@ -208,6 +208,40 @@
         <Map Name="Merida/ASMGCS_MMMD_LABELS" />
       </Maps>
     </Position>
+    <Position Name="MDA - Aproximación Merida" Type="ASD" DefaultCenter="+205613.280-0893927.700" DefaultRange="300" MagneticVariation="0">
```
Cambiar al esquema de nombres como Mexico. Empezando con la posición y al final el codigo de 3 letras

hatoriganzo commented on Sectors.xml:
```diff
@@ -35,6 +35,12 @@
+  <Sector FullName="Aproximación Merida" Frequency="121.200" Callsign="MMMD_APP" Name="MDA">
+    <Volumes>MMMD_TMA</Volumes>
+  </Sector>
+    <Sector FullName="Torre Merida" Frequency="118.300" Callsign="MMMD_TWR" Name="MDT">
```
Tabulacion

hatoriganzo reviewed (changes requested):
Wicho, apoyame con esos cambios porfa antes de hacer el merge.

wichoesparza committed:
Correcciones MID

wichoesparza committed:
Correcciones MID

Aqui se corrige la tabulacion en Sectors.xml y el nombre de la posicion de Tijuana a "Aproximacion Tijuana -  TJA"

wichoesparza committed:
Correccion nombres de posiciones en MZT ACC

wichoesparza committed:
Correccion nombres de posiciones en MZT ACC

wichoesparza committed:
Correcciones MID #2

Perdón, olvide guardar los cambios cuando hice el push anterior. Aquí se corrige el nombre de la posición a "Aproximación Mérida - MDA"

wichoesparza committed:
Merge remote-tracking branch 'origin/MMMDMerida' into MMMDMerida

gvaldezsan commented:
Se integra. Gracias Wicho!
